src/shapes/Transform.js

```diff
@@ -9,6 +9,11 @@ const transformOptions = {
     title: 'Number of loops',
     min: 1
   },
+  transformFrequency: {
+    title: 'Transform after each',
+    type: 'dropdown',
+    choices: ['point', 'loop'],
```

Comment:
What if instead of choices, we have just a toggle or checkmark. I am thinking it could be something like "Smear" (false being the new way) or "Keep Shape".

You're choice, and the question you're asking is more correct from a math perspective, but it forces people to think that way.

Reply:
We use "number of loops" already, so I think it's okay to keep that part. How about "Apply transforms [continuously, after each loop]"?

Reply:
Just to be clear, I am talking about the UI look of this feature, not the code names. I probably commented this in the wrong spot.

I'm really on the fence. I like to describe these as universally as I can, even at the expense of correctness. My inclination is to use the correct terms, but I get feedback like, "I used sandify with my X year old kids (or my XY year old parents), and they love how simple it is". 

Something like "smear" seems to describe the effect well, to me, at least.

Reply:
How about "When transforming" toggle: [Smear shape|Keep shape intact]?

I know "intact" is not a "kid" word, but it feels better to me.

Reply:
That seems better to me. I think "intact" is fine.

Reply:
> If I choose the "loop" option, and I choose only spin and not grow, it only creates the first and second rotations. Even though the number of loops it much higher.

This is fixed.

Reply:
This has been changed.